test/runtests.jl

```diff
@@ -10,7 +10,7 @@ struct Foo end
 # if code has no calls then it must be fully static
 has_no_calls(ir) = all(stmt->!Meta.isexpr(stmt, :call), ir)
 
-@testset "static_hasmethod" begin
+VERSION >= v"1.3" && @testset "static_hasmethod" begin
```

Comment:
I think you should use normal if here 🤔 or use paranteces around `@testset`
```suggestion
if VERSION >= v"1.3" 
	@testset "static_hasmethod" begin
```

Reply:
I disagree, the extra indent adds nothing and this is a common enough pattern for testsets.

Reply:
I thought the parser has some issues parsing this. Turns out it was another sentence.